<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordTeachingDialogClass</class>
 <widget class="QDialog" name="RecordTeachingDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>脚本录制</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="currentScriptTypeLabel">
         <property name="text">
          <string>当前脚本类型</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="currentScriptTypeLineEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="teachingTextEdit"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QCheckBox" name="ifShowCheckBox">
       <property name="text">
        <string>下次不在展示</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="exitPushButton">
       <property name="text">
        <string>退出录制</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextStepPushButton">
       <property name="text">
        <string>进入录制（Enter）</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
